scripts/_common.sh

```diff
@@ -3,140 +3,23 @@
 # Common variables and helpers
 #
 
-pkg_dependencies="php7.0-fpm sipcalc dnsutils openvpn curl fake-hwclock"
+pkg_dependencies="php-fpm sipcalc dnsutils openvpn curl fake-hwclock"
```

Comment:
Uh but this is going to trigger weird issues like, for example if the Sury repository is installed (this is where newer versions of php are from) then requiring php-fpm will in fact install php-fpm 7.4 ...

It's really better to have an explicit version requirement... Note that it's not really an issue to still depend on 7.0 because Yunohost automagically patches everything to depend on 7.3 instead. (well, except if there are some hidden places which refer to php7.0)

Reply:
you're right, and since I'm unable to make tests at the moment I'll revert these changes